media-sound/beets/beets-1.4.9-r3.ebuild

```diff
@@ -22,49 +22,48 @@ else
 fi
 
 DESCRIPTION="Media library management system for obsessive-compulsive music geeks"
-HOMEPAGE="http://beets.io/ https://pypi.org/project/beets/"
+HOMEPAGE="https://beets.io/ https://pypi.org/project/beets/"
 
 LICENSE="MIT"
 SLOT="0"
-IUSE="badfiles chromaprint discogs doc ffmpeg gstreamer icu lastfm mpd replaygain test thumbnail webserver"
+IUSE="badfiles chromaprint cors discogs doc ffmpeg gstreamer icu lastfm mpd replaygain test thumbnail webserver"
 
-RDEPEND="${DEPEND}"
-DEPEND="
+RDEPEND="
 	$(python_gen_cond_dep '
-		test? (
-			dev-python/wheel[${PYTHON_MULTI_USEDEP}]
-			dev-python/beautifulsoup[${PYTHON_MULTI_USEDEP}]
-			dev-python/flask[${PYTHON_MULTI_USEDEP}]
-			dev-python/mock[${PYTHON_MULTI_USEDEP}]
-			dev-python/rarfile[${PYTHON_MULTI_USEDEP}]
-			dev-python/responses[${PYTHON_MULTI_USEDEP}]
-			dev-python/pyxdg[${PYTHON_MULTI_USEDEP}]
-		)
 		>=dev-python/jellyfish-0.7.1[${PYTHON_MULTI_USEDEP}]
 		dev-python/munkres[${PYTHON_MULTI_USEDEP}]
+		>=media-libs/mutagen-1.33[${PYTHON_MULTI_USEDEP}]
 		>=dev-python/python-musicbrainz-ngs-0.4[${PYTHON_MULTI_USEDEP}]
 		dev-python/pyyaml[${PYTHON_MULTI_USEDEP}]
-		dev-python/requests[${PYTHON_MULTI_USEDEP}]
 		dev-python/requests-oauthlib[${PYTHON_MULTI_USEDEP}]
+		dev-python/requests[${PYTHON_MULTI_USEDEP}]
 		>=dev-python/six-1.9[${PYTHON_MULTI_USEDEP}]
 		dev-python/unidecode[${PYTHON_MULTI_USEDEP}]
-		>=media-libs/mutagen-1.33[${PYTHON_MULTI_USEDEP}]
 		badfiles? (
 			media-libs/flac
 			media-sound/mp3val
 		)
 		chromaprint? (
+			dev-python/audioread[${PYTHON_MULTI_USEDEP}]
```

Comment:
I don't think this is correct. Beets doesn't directly use audioread, pyacoustid does and dev-python/pyacoustid already depends on audioread.
(same for the 9999 ebuild)

Reply:
I added this with 0e2824ee4b35816f16a2c661c2d2daaeaed4f1c1 regarding https://bugs.gentoo.org/693164
but as dev-python/audioread is a direct dependency of dev-python/pyacoustid it seems fine to trim this off.

Reply:
Yeah, I think it can be removed. I've added a link to a search in the beets codebase to the previous comment as well so you can check.

Reply:
Hey @simonvanderveldt thank you for the feedback,
the dependency 'dev-python/audioread' as been dropped as it is a provided by 'dev-python/pyacoustid'.